src/functions-and-arrays.js

```diff
@@ -78,7 +141,21 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+
+function howManyTimes (a,b){
```

Comment:
`a` and `b` are not good names for the arguments, because they are too generic. Something like `words` and `wordToSearchFor` would be better